<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>821</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="comboBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>0</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>과목번호</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>교과목명</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>담당교수</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>개설학과</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>조회</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="lect_table">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>821</width>
     <height>361</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>160</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>과목번호</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>교과목명</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>담당교수</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>강의시간</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>개설학과</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="EnrollmentButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>수강신청</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
